CoreFoundation/Base.subproj/ForSwiftFoundationOnly.h

```diff
@@ -484,6 +486,25 @@ static inline unsigned int _dev_minor(dev_t rdev) {
 }
 #endif
 
+// ICU function can be renamed on some platforms eg on Linux udtivfmt_open() -> udtitvfmt_open_61_swift() so wrap the functions
+// to provide a platform independant interface. These functions are used by DateIntervalFormatter.swift
```

Comment:
'independent' but also not sure what the comment about means... we can use version-specific ICU functions? But we are not?

Reply:
The ICU libraries  on Linux are built using the `--enable-renaming --with-library-suffix=swift` options which renames functions to include the library version and optional suffix to avoid symbol clash  allowing multiple versions of the library to be linked in at the same time

Reply:
Ah okay thanks for the explanation
Didn't/not seeing the udtitvfmt_open_61_swift call but I suppose it's macro'd somewhere... not obvious online :-)

Reply:
Yeah, there is a `<unicode/urename.h>` that  has 
```
#define udtitvfmt_close U_ICU_ENTRY_POINT_RENAME(udtitvfmt_close)
#define udtitvfmt_format U_ICU_ENTRY_POINT_RENAME(udtitvfmt_format)
#define udtitvfmt_open U_ICU_ENTRY_POINT_RENAME(udtitvfmt_open)
``` 

etc